New pull request MathExp-kpetrov2

kpetrov2 opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

<!-- Please check ALL the boxes: -->

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=1676) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=1676) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: kpetrov2<br/>:white_check_mark: SSwiniarski<br/>:white_check_mark: yangc95<br/>:x: awgraves<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=1676) it.</sub>

kpetrov2 commented:
Hi @yangc95 ! I've changed the LaTex code line with an image.
I have set my username to be public and added my GitHub profile, but I think the problem persists. Should I make any more changes? I attach a picture to see my configuration.
![contributor issue](https://user-images.githubusercontent.com/72105079/219855794-154f2e27-174a-4071-bfbc-3b662a88bd8c.jpg)

SSwiniarski commented on content/c-sharp/concepts/math-functions/terms/exp/exp.md:
```diff
@@ -17,9 +17,7 @@ The **`Math.Exp()`** method returns the value of raising `e` to the power of a g
-$$
-\sum\limits_{n=0}^{\infty} \frac{1}{n!} = 1 + \frac{1}{n+1} + \frac{1}{(n+1)(n+2)} + \frac{1}{(n+1)(n+2)(n+3)} + \cdots
-$$
+![eFormula](https://user-images.githubusercontent.com/72105079/219855529-5cb30211-ab0d-45ca-bd19-25c36386f83c.jpg)
```
The image file needs to be hosted in the [/media folder](https://github.com/Codecademy/docs/tree/main/media).
Use the following url to link to it: https://raw.githubusercontent.com/Codecademy/docs/main/media/image-name.jpg.
Change the name of the image to something like "e-formula.jpg"

SSwiniarski reviewed (changes requested):
@kpetrov2 I reviewed this and you need to make a few changes.

SSwiniarski commented:
@kpetrov2, you may have to close this PR and start a new one making sure you're using your Github user id. The CLA is seeing an extra user, awgraves, attached to this PR.
